### PR TITLE
make upstream tests as separate make target

### PR DESCRIPTION
**Pull request by zmiklank**
\+ small fix in a separate commit

Setting as draft for now.

### CODE REVIEW

**Comment by zmiklank** [comment]
Depends on: https://github.com/sclorg/container-common-scripts/pull/352

**Comment by zmiklank** [comment]
[test]

File: Makefile [hunk 1]
```diff
@@ -8,3 +8,7 @@ OPENSHIFT_NAMESPACES =
 .PHONY: $(shell test -f common/common.mk || echo >&2 'Please do "git submodule update --init" first.')
 
 include common/common.mk
+
+.PHONY: test-upstream-unittests
```
Comment (phracek) [inline Makefile:12]: I would suggest the name `test-upstream` w/o unittests

**Review by phracek — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (zmiklank) [reply Makefile:12]: Sure.

Comment (zmiklank) [reply Makefile]: Fixed with [f42a446](https://github.com/sclorg/s2i-nodejs-container/pull/415/commits/f42a446d1d17b54cb90d77c13bfb363092e20835).

**Review by phracek — APPROVED** [review]
LGTM. Thanks for separation upstream tests.

**Comment by zmiklank** [comment]
[test]

**Comment by zmiklank** [comment]
Test failures are caused from within the `nodejs-app` - it is not related to this PR. Explanation can be seen here: https://github.com/sclorg/nodejs-ex/pull/251#issuecomment-1845658792

**Pull request merged**